src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Read.Helpers.cs

```diff
@@ -39,8 +39,7 @@ private static TValue ReadCore<TValue>(JsonConverter jsonConverter, ref Utf8Json
 
         internal static bool IsValidNumberHandlingValue(JsonNumberHandling handling)
```

Comment:
This is in `JsonSerializer.Read.Helpers.cs` file but number handling is not exclusively to deserialization.
Can this be in `JsonHelpers` class instead?

Reply:
I created a new `JsonSerializer.cs` file, so that we don't end up adding multiple serializer-only helpers to `JsonHelpers`.